packages/plugins/solidity/src/solcP.js

```diff
@@ -21,9 +21,13 @@ class SolcProcess extends ProcessWrapper {
     // TODO: only available in 0.4.11; need to make versions warn about this
     try {
       let func = this.solc.compileStandardWrapper;
-      if (semver.gte(this.solc.version(), '0.5.0')) {
+      const solcVersion = this.solc.version();
+      if (semver.gte(solcVersion, '0.5.0')) {
         func = this.solc.compile;
       }
+      if (semver.gte(solcVersion, '0.6.0')) {
```

Comment:
Very unimportant, but you could move this if above the other and use `else if` for the other (one less condition evaluation 🚀 )

Reply:
Nice suggestion! Done.